ohmec_data_ma.js

```diff
@@ -4,8 +4,8 @@ dataNA = {
   "_comment3":"SPDX-License-Identifier: Apache-2.0",
   "type":"FeatureCollection",
   "viewpoint": {
-    "startdatestr":"1600:01:01",
-    "curdatestr":  "1764:01:01",
+    "startdatestr":"10000:01:01BC",
+    "curdatestr":  "600:01:01",
     "enddatestr":  "2010:12:31",
```

Comment:
Go ahead and make `enddatestr` = 1600, which is where we'll (hopefully) tie into the main map